COMMUNITY.md

```diff
@@ -13,13 +13,43 @@ As opposed to defining a big overarching burden of responsibilities the team wou
   - Slack
 - Oversight of [this repository](https://github.com/fluxcd/community).
 
+### Communications
+
+We as a project want to make it a priority to inform our community and the wider world of developments well. It's what helps new contributors get up to speed, enables collaboration, integrations and more.
```

Comment:
I'd like to suggest we adopt one sentence per line for markdown files, per the argument at https://sembr.org/. Not a requirement, just a suggestion. I had applied this for the GOVERNANCE and SECURITY md files, so maybe we can adopt that standard for this community repo, and discuss separately whether or not to adopt this standard across other fluxcd repos?

```suggestion
We as a project want to make it a priority to inform our community and the wider world of developments well.
It's what helps new contributors get up to speed, enables collaboration, integrations and more.
```